### DISCUSSION TITLE
.25 chia missing

### DISCUSSION THREAD

**Discussion by EVILTIM58** [Chia-Network/chia-blockchain#8186] [Support]
I amazingly found a block on my third day.  However the .25 chia never arrived.  WTF

**Comment by EVILTIM58** [comment]
looking at chia explorer, it says it paid out to my address, but I dont have it, nor any record of it inbound.   I solved block 638465..  anyone have any ideas or see something I am missing?

**Reply by sixym3** [reply]
Yea same thing happening here, my wallet is synced and I also farmed a block, but I did not get the 0.25 xch rewards.

**Comment by EVILTIM58** [comment]
![image](https://user-images.githubusercontent.com/89432385/130975898-99f2622e-b651-431a-be71-c6d2fbadd1bc.png)

**Comment by BrandtH22** [comment]
@EVILTIM58 , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
